### PR TITLE
1.5.x fixes

### PR DESCRIPTION
**Pull request by zvin**
Change-type: patch

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "balena-etcher",
-  "version": "1.5.0",
+  "version": "1.5.2",
```
Comment (lurch) [inline npm-shrinkwrap.json]: 1.5.2 and not 1.5.1 ?

Comment (zvin) [reply npm-shrinkwrap.json]: This file is generated, the version is copied from the `package.json` file which has `1.5.2` on master.

Comment (lurch) [reply npm-shrinkwrap.json]: Ahhh, I see from https://github.com/balena-io/etcher/commits/master that **resin-ci** is now bumping the version every time a PR gets merged.

File: lib/gui/app/os/dropzone/directives/dropzone.js [hunk 2]
```diff
@@ -51,18 +51,21 @@ module.exports = ($timeout) => {
...
-        // Safely bring this to the word of Angular
-        $timeout(() => {
-          scope.osDropzone({
+        if (event.dataTransfer.files.length) {
+          const filename = _.first(event.dataTransfer.files).path
 
-            // Pass the filename as a named
-            // parameter called `$file`
-            $file: filename
+          // Safely bring this to the word of Angular
```
Comment (lurch) [inline lib/gui/app/os/dropzone/directives/dropzone.js]: Is `word` a typo for `world`?

Comment (zvin) [reply lib/gui/app/os/dropzone/directives/dropzone.js]: I guess

Comment (zvin) [reply lib/gui/app/os/dropzone/directives/dropzone.js]: fixed

File: lib/gui/app/modules/image-writer.js [hunk 3]
```diff
@@ -392,7 +392,9 @@ exports.cancel = () => {
 
   try {
     const [ socket ] = ipc.server.sockets
-    ipc.server.emit(socket, 'cancel')
+    if (socket !== undefined) {
```
Comment (zrzka) [inline lib/gui/app/modules/image-writer.js]: Should we add ...

```
// eslint-disable-next-line no-undefined
```

... to make linter happy? Same "comment" is in other parts of the source code.

Comment (zvin) [reply lib/gui/app/modules/image-writer.js]: updated

**Pull request merged**